arangod/Pregel/Conductor.cpp

```diff
@@ -953,16 +938,12 @@ ErrorCode Conductor::_sendToAllDBServers(std::string const& path, VPackBuilder c
     } else {
       TRI_ASSERT(SchedulerFeature::SCHEDULER != nullptr);
       Scheduler* scheduler = SchedulerFeature::SCHEDULER;
-      bool queued =
-          scheduler->queue(RequestLane::INTERNAL_LOW, [this, path, message,
-                                                       self = shared_from_this()] {
-            TRI_vocbase_t& vocbase = _vocbaseGuard.database();
-            VPackBuilder response;
-            _feature.handleWorkerRequest(vocbase, path, message.slice(), response);
-          });
-      if (!queued) {
-        return TRI_ERROR_QUEUE_FULL;
-      }
+      scheduler->queue(RequestLane::INTERNAL_LOW, [this, path, message,
+                                                   self = shared_from_this()] {
+        TRI_vocbase_t& vocbase = _vocbaseGuard.database();
+        VPackBuilder response;
+        _feature.handleWorkerRequest(vocbase, path, message.slice(), response);
+      });
     }
     return TRI_ERROR_NO_ERROR;
```

Comment:
It looks like this method now always returns `TRI_ERROR_NO_ERROR`. Shall we declare it void?

Reply:
Only in the cluster case, in the single server case we also have code paths that can return error codes.